superset/charts/schemas.py

```diff
@@ -718,6 +718,7 @@ class ChartDataQueryObjectSchema(Schema):
     )
     groupby = fields.List(
         fields.String(description="Columns by which to group the query.",),
+        allow_none=True,
```

Comment:
`QueryObject` automatically does `self.groupby = groupby or []` in the constructor, so no need to require a `fields.List`.